realis.go

```diff
@@ -75,6 +75,8 @@ type Realis interface {
 	MaintenanceStatus(hosts ...string) (*aurora.Response, *aurora.MaintenanceStatusResult_, error)
 	SetQuota(role string, cpu *float64, ram *int64, disk *int64) (*aurora.Response, error)
 	GetQuota(role string) (*aurora.Response, error)
+	Snapshot() error
```

Comment:
Actually I prefer having comment on the interface, godoc would look better and normally when people jump to definition, the first jump to the interface.

Reply:
Good point, will tackle this in a different PR where I'll move all comments to the interface.